src/components/scheduletable.js

```diff
@@ -0,0 +1,68 @@
+import React, {Component} from 'react';
+import {View, ActivityIndicator} from 'react-native';
+import {ThemedText} from '../components/components';
+import BuildConfigs from '../config';
+import ForestApi from '../tools/apis';
+
+export default class ScheduleTable extends Component{
+  constructor(props){
+    super(props);
+    this.state = {
+      dates: [],
+      contents: [],
+      isLoading: false
+    };
+  }
+  async componentDidMount(){
+    this.setState({isLoading: true});
+    let schedule = await ForestApi.post('/life/schedules', JSON.stringify({
+      'year': this.props.year,
+      'month': this.props.month
+    }), false);
+    if(schedule.ok){
+      let data = await schedule.json();
+      let dates = [], contents = [];
+      for(let item of data.schedules){
+        dates.push(`${item.period}`);
+        contents.push(` | ${item.content}`);
```

Comment:
```suggestion
        contents.push(` ${item.content}`);
```
작업된 결과물 사진 보니 `|` 표시 굳이 없어도 괜찮을듯